docs/user-guide/reference/conditional-forwarding.md

```diff
@@ -2,31 +2,32 @@
 
 After processing the data, conditional forwarding is doable based on the `Tags` returned in the result. 
 Below is list of different logic operations that can be done on tags.
-- **and** - forwards the message if all the tags specified are present in Message's tags.
-- **or** - forwards the message if one of the tags specified is present in Message's tags.
-- **not** - forwards the message if all the tags specified are not present in Message's tags.
+
+* **and** - forwards the message if all the tags specified are present in Message's tags.
+* **or** - forwards the message if one of the tags specified is present in Message's tags.
+* **not** - forwards the message if all the tags specified are not present in Message's tags.
 
 For example, there's a UDF used to process numbers, and forward the result to different vertices based on the number is even or odd. In this case, you can set the `tag` to `even-tag` or `odd-tag` in each of the returned messages,
 and define the edges as below:
 
 ```yaml
 edges:
   - from: p1
-    to: even-vertex
+    to: vertex-1
```

Comment:
I understand this example is to showcase `or`, `and` and `not`, but somehow it does not make sense.